README.md

```diff
@@ -261,7 +262,7 @@ Input parameter - object of type `Service`, `Pod`, `ReplicationController`.
 The below example is for v1
 
 ```ruby
-service = Service.new
+service = Kubeclient::Resource.new
```

Comment:
what makes it a service then?  do you need `.kind`?  or is it implicit in passing it to `.create_service()` in the end?
IOW, have you verified the example works?